keyboard/src/lib.rs

```diff
@@ -1,11 +1,433 @@
 #![no_std]
 
-static CODES: [u8; 59] = *b"??1234567890-=??qwertyuiop[]\n?asdfghjkl;'`?\\zxcvbnm,./?*? ?";
+pub enum Action {
+    Pressed,
+    Released,
+}
+
+#[derive(Clone, Copy, Debug, PartialEq)]
+pub enum Key {
+    KA,
```

Comment:
This is mainly for the plain digits below (`0`, `1`, etc.). I can remove the `K` prefix and just do `Digit0`, `Digit1`, etc. for those. How does that approach sound to you?